labels.json

```diff
@@ -1561,6 +1571,11 @@
                 "lang": "en",
                 "name": "Raptor",
                 "description": "This user is a Raptor! [Category: Dinosaur]"
+            },
+             {
+                "lang": "pt-BR",
+                "name": "Velociraptor",
+                "description": "Este usuário é um Velociraptor! [Categoria: Dinossauro]"
```

Comment:
Please revert "Categoria" to "Category" (this is used on the sonasky-browse site for parsing; Dinossauro is fine, but "Category" must remain.

Reply:
Okay! I'll revert it